### ISSUE TITLE
Tutorial problems - Entrez

### ISSUE THREAD

**Issue by tiagoantao** [biopython/biopython#750]
Problems on the Entrez chapter:
1. 

``` python
if not os.path.isfile(filename):
    # Downloading...
    net_handle = Entrez.efetch(db="nucleotide",id="186972394",rettype="gb", retmode="text")
```

Raises

AttributeError: **exit**

**Comment by peterjc** [comment]
I can't reproduce this, the example seems to work fine under Python 2.6, 2.7, 3.3, 3.4, 3.5

Which version of Python was used? My hunch is a standard library context handler bug might be to blame, which could have been fixed in a point release.

As an aside, I fixed the PEP8 spacing in this example: 95a1cae6224f7ea5e63c1bfa550555ebd251c468

**Comment by peterjc** [comment]
I'm going to close this since I couldn't reproduce it - I suspect it was fixed in a Python 2.x point release.
